lib/elastic_model/acts_as_elastic_model.rb

```diff
@@ -259,7 +259,9 @@ def bulk_index(batch, options = { })
             refresh: options[:wait_for_index_refresh] ? "wait_for" : false
           })
           if batch && batch.length > 0 && batch.first.respond_to?(:last_indexed_at)
-            where(id: batch).update_all(last_indexed_at: Time.now)
+            ActiveRecord::Base.connection.without_sticking do
+              where(id: batch).update_all(last_indexed_at: Time.now)
+            end
```

Comment:
Presumably the UPDATE gets run on the primary, but what are the subsequent queries that need to be "unstuck" and run on the replica?

Reply:
When we index an obs, we'll update it's last_indexed_at time. But we pretty much never need to turn around and query for that attribute in a time frame longer than expected replication lag. So this is making sure that for any request that indexes an observation, it's not stuck querying the primary for the duration of the request after doing the indexing, so we can spread around the post-indexing query load. Where we stick and don't will need some tweaking as we see how this works and how well load is split between the DBs